app-typescript/components/Radio.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+
+interface IValue {
+    value: any;
+    text: string;
+    disabled?: boolean;
+}
+
+interface IProps {
+    model: any;
+    values: Array<IValue>;
+    side?: string;
+    onClick(nextValue: string): void;
```

Comment:
Use `onChange` which is a more standard name. Most inputs can be changed via keyboard too.